pkg/controller/pxc/controller.go

```diff
@@ -218,19 +219,19 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(ctx context.Context, request r
 		for _, fnlz := range o.GetFinalizers() {
 			var sfs api.StatefulApp
 			switch fnlz {
-			case "delete-ssl":
+			case naming.FinalizerDeleteSSL:
 				err = r.deleteCerts(o)
-			case "delete-proxysql-pvc":
+			case naming.FinalizerDeleteProxysqlPvc:
 				sfs = statefulset.NewProxy(o)
 				// deletePVC is always true on this stage
 				// because we never reach this point without finalizers
 				err = r.deleteStatefulSet(o, sfs, true, false)
-			case "delete-pxc-pvc":
+			case naming.FinalizerDeletePxcPvc:
 				sfs = statefulset.NewNode(o)
 				err = r.deleteStatefulSet(o, sfs, true, true)
 			// nil error gonna be returned only when there is no more pods to delete (only 0 left)
 			// until than finalizer won't be deleted
-			case "delete-pxc-pods-in-order":
+			case naming.FinalizerDeletePxcPodsInOrder:
```

Comment:
I think we can't just stop supporting old finalizers, we need to support them both. It'd be nice if we log that they're deprecated if user still uses the old ones

Reply:
I'm a bit confused. For me it looks like I just moved the value to the variable, not deletion. 
How do you think it is the best way to fix it?